projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```
@@ -286,19 +290,17 @@
 
   .monthpicker,
   .yearpicker,
-  .years {
+  .daypicker {
     display: flex;
-    flex-direction: column;
     flex-wrap: wrap;
+    row-gap: tokens.$cds-global-space-5;
+    height: calc(calc((9 * var(--cds-global-space-10))) + calc(3 * var(--cds-global-space-5)));
```

Use tokens for calculation

```suggestion
    height: calc(calc(9 * tokens.$cds-global-space-10) + calc(3 * tokens.$cds-global-space-5));
```

---

Fixed in [948b80d](https://github.com/vmware-clarity/ng-clarity/pull/1706/commits/948b80d3d202cba561298fdb5a6636cfd8e84a57)